TeaDiary/Scripts/App/Teas/Details.js

```diff
@@ -12,7 +26,17 @@
                 redirectToTeaList();
             }
         });
-});
+}
+
+function createPageLoaded() {
+    $("#Update").hide();
+    $("#Delete").hide();
+
+    $("#Name").val("");
+    $("#AdditionalName").val("");
+    $("#Type").val("");
+    $("#Notes").val("");
```

Comment:
Why don't you also encapsulate this input management in the tea displayer? It can support these operations: displayById, displayEmpty, getTeaModel (this one you can use in your `onCreate/UpdateClick()` to better separate the concerns)